pkg/services/app/add_test.go

```diff
@@ -359,5 +359,32 @@ var _ = Describe("New directory structure", func() {
 		Expect(r[0].(string)).To(Equal("../../../apps/" + origName))
 		Expect(r[1].(string)).To(Equal("../../../apps/" + addParams.Name))
 	})
+	It("deals with a cluster dir with additional subdirectories", func() {
```

Comment:
Shall we add a line break between the `It` assertions? Unrelated to these changes but I noticed that this file is not consistent in terms of the line breaks.